pom.xml

```
@@ -83,8 +83,8 @@
         <failOnMissingWebXml>false</failOnMissingWebXml>
 
         <!-- Dependencies -->
-        <spring.version>5.3.17</spring.version>
-        <spring.boot.version>2.6.5</spring.boot.version>
+        <spring.version>5.3.18</spring.version>
+        <spring.boot.version>2.6.6</spring.boot.version>
```

```suggestion
        <spring.boot.version>2.6.5</spring.boot.version>
```

Let's change Spring version for now until Spring Boot is released.

---

Boot is going to be released today

---

There is no point in only upgrading Spring

---

I was thinking just to see if the tests pass right away. But probably they will anyway.

---

Only fusion-endpoint uses the spring version as everything else is defined through the `<artifactId>spring-boot-dependencies</artifactId>` bom using the spring.boot.version